packages/openapi3-parser/lib/parser/oas/parseSchemaObject.js

```diff
@@ -192,7 +192,7 @@ function parseType(context) {
     ensureTypesAreUnique,
 
     // FIXME support >1 type
-    R.when(e => e.length > 1, createWarning(namespace, `'${name}' 'type' more than one type is current unsupported`)));
+    R.when(e => e.length > 1 && !e.contains('null'), createWarning(namespace, `'${name}' 'type' more than one type is current unsupported`)));
```

Comment:
['string', 'boolean', 'null'] would fail this

Reply:
Very good point, I've applied this update:

```diff
diff --git a/packages/openapi3-parser/lib/parser/oas/parseSchemaObject.js b/packages/openapi3-parser/lib/parser/oas/parseSchemaObject.js
index 1e3ac22..7dd7595 100644
--- a/packages/openapi3-parser/lib/parser/oas/parseSchemaObject.js
+++ b/packages/openapi3-parser/lib/parser/oas/parseSchemaObject.js
@@ -192,7 +192,10 @@ function parseType(context) {
     ensureTypesAreUnique,
 
     // FIXME support >1 type
-    R.when(e => e.length > 1 && !e.contains('null'), createWarning(namespace, `'${name}' 'type' more than one type is current unsupported`)));
+    R.unless(
+      e => e.length === 0 || e.length === 1 || e.length == 2 && e.contains('null'),
+      createWarning(namespace, `'${name}' 'type' more than one type is current unsupported`)
+    ));
 
   return R.cond([
     [isString, parseStringType],
```

Normally I'd make a test for that case, but.. I am preparing a next PR that makes it supported so unless there's push back I'll not add a negative test for that case.